### PR TITLE
fix cadecy issue: no assertions

### LINKED ISSUE #296: [First Timers Only] Add assert to shouldReturnNothingWhenIsNotHydrateCommand unit test
**Note: This issue is to be picked up by first time contributors to this project only. If it is not picked up after ~7 days, it will be opened up for anyone to take.**

## Task Context

The Codacy Static Code Analysis system is reporting an issue coming from `CommandInvokerTest.java`. The reported issue can be found at:

https://app.codacy.com/gh/jmakhack/hydrate-reminder/issues?&filters=W3siaWQiOiJMYW5ndWFnZSIsInZhbHVlcyI6W119LHsiaWQiOiJDYXRlZ29yeSIsInZhbHVlcyI6WyJDb2RlU3R5bGUiXX0seyJpZCI6IkxldmVsIiwidmFsdWVzIjpbXX0seyJpZCI6IlBhdHRlcm4iLCJ2YWx1ZXMiOlsiNTU3NSJdfSx7ImlkIjoiQXV0aG9yIiwidmFsdWVzIjpbXX1d

## Acceptance Criteria / Goals

The goal of this task is to fix the issue(s) reported by Codacy by making adding an assertion to `shouldReturnNothingWhenIsNotHydrateCommand`.

## Additional Notes

After opening a pull request, the Codacy Static Code Analysis will automatically run. It should report that the fixed issue(s) if this task is completed successfully.

Please refer to [CONTRIBUTING.md](https://github.com/jmakhack/hydrate-reminder/blob/master/CONTRIBUTING.md) for info on how to setup, build, run, and test this project. Feel free to ping @jmakhack in the comments below with any questions if needed. Also, join the [Discord](https://discord.gg/RTgxfFW9mS) to meet and engage with other contributors!

[](https://discord.gg/RTgxfFW9mS)

### PR DESCRIPTION
**Pull request by NNRepos**
## Associated Issue
Closes #296 

## Implemented Solution
change test functionality, to count calls instead of asserting return value.
(void must return nothing anyway).

### CODE REVIEW

File: src/test/java/com/hydratereminder/command/CommandInvokerTest.java [hunk 1]
```diff
@@ -83,9 +83,11 @@ public void shouldReturnNothingWhenIsNotHydrateCommand() {
         // given
         CommandExecuted commandToExecute = new CommandExecuted("aa", new String[]{"hydrate"});
 
-        // when and then
-        commandInvoker.invokeCommand(commandToExecute);
+        // when
+        final Object result = commandInvoker.invokeCommand(commandToExecute);
```
Comment (macslowiak) [inline src/test/java/com/hydratereminder/command/CommandInvokerTest.java:87]: Is this test working ? 
Return type of invokeCommand is void.

Comment (NNRepos) [reply src/test/java/com/hydratereminder/command/CommandInvokerTest.java]: i expected the tests to be run by git actions, my bad.
i'm used to python 😅

File: src/test/java/com/hydratereminder/command/CommandInvokerTest.java [hunk 2]
```diff
@@ -79,13 +79,15 @@ public void shouldCallCommandCreatorTwiceWhenNotSupportedCommandExceptionIsThrow
...
+    public void shouldNotCallCommandCreatorWhenIsNotHydrateCommand() {
         // given
         CommandExecuted commandToExecute = new CommandExecuted("aa", new String[]{"hydrate"});
 
-        // when and then
+        // when
         commandInvoker.invokeCommand(commandToExecute);
 
+        // then
+        verify(CommandCreator, times(0)).createFrom(any());
```
[the remark this answers is not in the document]
Comment (NNRepos) [reply src/test/java/com/hydratereminder/command/CommandInvokerTest.java]: thanks, fixed

**Review by macslowiak — APPROVED** [review]

**Comment by mslowiak** [comment]
@NNRepos sorry, we cannot proceed with your PR since another contributor is working on that.
Firstly, we ask under issue if it can be picked up, then we can start work - just for the future :) 

cc: @jmakhack

**Comment by NNRepos** [comment]
ok, i didn't realize that's how it works.
i will ask for permission on a different issue :)

**Pull request closed without merging**